src/parameters/path/user_id.yaml

```diff
@@ -1,6 +1,6 @@
 name: user_id
 in: path
 required: true
-description: The id of the user to retrieve
+description: The id of an user
```

Comment:
```suggestion
description: The id of a user
```
gottem https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

Reply:
This is madness